<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BigRenderCtrl</class>
 <widget class="QWidget" name="BigRenderCtrl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>746</width>
    <height>60</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BigRenderCtrl</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>20</number>
      </property>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_teacher">
        <property name="styleSheet">
         <string notr="true">font: 9pt &quot;微软雅黑&quot;;
color: rgb(153, 153, 153);</string>
        </property>
        <property name="text">
         <string>老师</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_LocalCamera">
        <property name="minimumSize">
         <size>
          <width>42</width>
          <height>42</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>42</width>
          <height>42</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">
QPushButton#pushButton_LocalCamera{
border-image: url(:/big/img/ctrl/big/big_render_camera_open_Normal.png);
}
QPushButton#pushButton_LocalCamera:hover{
border-image: url(:/big/img/ctrl/big/big_render_camera_open_Click.png);
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_LocalMic">
        <property name="minimumSize">
         <size>
          <width>42</width>
          <height>42</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>42</width>
          <height>42</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton#pushButton_LocalMic{
border-image: url(:/big/img/ctrl/big/big_mic_Normal.png);
}
QPushButton#pushButton_LocalMic:hover{
border-image: url(:/big/img/ctrl/big/big_mic_Click.png);
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_fullScreen">
        <property name="minimumSize">
         <size>
          <width>42</width>
          <height>42</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>42</width>
          <height>42</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton#pushButton_fullScreen{
border-image: url(:/big/img/ctrl/big/big_render_fullScreen_Normal.png);
}
QPushButton#pushButton_fullScreen:hover{
border-image: url(:/big/img/ctrl/big/big_render_fullScreen_Click.png);
}
</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_changeCamera">
        <property name="minimumSize">
         <size>
          <width>42</width>
          <height>42</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>42</width>
          <height>42</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">

QPushButton#pushButton_changeCamera{
border-image: url(:/big/img/ctrl/big/changeCameraBig_Normal.png);
}
QPushButton#pushButton_changeCamera:hover{
border-image: url(:/big/img/ctrl/big/changeCameraBig_Click.png);
}
</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_big_screen">
        <property name="minimumSize">
         <size>
          <width>42</width>
          <height>42</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>42</width>
          <height>42</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
border-image: url(:/big/img/ctrl/big/big_change_big_screen.png);
}
QPushButton:hover{
border-image: url(:/big/img/ctrl/big/big_change_big_screen_hover.png);
}

</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_stu">
        <property name="styleSheet">
         <string notr="true">font: 9pt &quot;微软雅黑&quot;;
color: rgb(153, 153, 153);</string>
        </property>
        <property name="text">
         <string>学生</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_AllMuteMic">
        <property name="minimumSize">
         <size>
          <width>42</width>
          <height>42</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>42</width>
          <height>42</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton#pushButton_AllMuteMic{
	border-image: url(:/big/img/ctrl/big/big_multi_mute_Normal.png);


}
QPushButton#pushButton_AllMuteMic:hover{
	border-image: url(:/big/img/ctrl/big/big_multi_mute_Click.png);
}
</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_AllUnPublish">
        <property name="minimumSize">
         <size>
          <width>42</width>
          <height>42</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>42</width>
          <height>42</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton#pushButton_AllUnPublish{
	
	border-image: url(:/big/img/ctrl/big/big_render_all_unpublish_Normal.png);
}
QPushButton#pushButton_AllUnPublish:hover{
border-image: url(:/big/img/ctrl/big/big_render_all_unpublish_Click.png);
}
</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_closeStuCamera">
        <property name="minimumSize">
         <size>
          <width>42</width>
          <height>42</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>42</width>
          <height>42</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">
QPushButton{
border-image: url(:/big/img/ctrl/big/big_render_camera_open_Normal.png);
}
QPushButton:hover{
border-image: url(:/big/img/ctrl/big/big_render_camera_open_Click.png);
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_closeStuMic">
        <property name="minimumSize">
         <size>
          <width>42</width>
          <height>42</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>42</width>
          <height>42</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
border-image: url(:/big/img/ctrl/big/big_mic_Normal.png);
}
QPushButton:hover{
border-image: url(:/big/img/ctrl/big/big_mic_Click.png);
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_UnpublishStu">
        <property name="minimumSize">
         <size>
          <width>42</width>
          <height>42</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>42</width>
          <height>42</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">

QPushButton{
border-image: url(:/big/img/ctrl/big/big_unpublish_Normal.png);
}
QPushButton:hover{
border-image: url(:/big/img/ctrl/big/big_unpublish_Click.png);
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../../VLive/VLive.qrc"/>
 </resources>
 <connections/>
</ui>
